src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

Comment:
DRY
move transition duration to to variable

also, specify effects you need to apply transition, because it adds to all ones by default